README.md

```diff
@@ -292,12 +136,4 @@ found a good way to handle its small screen.
 Also note that iOS supports `classList` and `dataset` APIs starting with version 5, so iOS 4.X and older
 requires polyfills to work.
 
-
-LICENSE
----------
-
-Copyright 2011-2012 Bartek Szopka
-
-Released under the MIT and GPL (version 2 or later) Licenses.
-
-
+Copyright 2011-2016 Bartek Szopka - [Licence](LICENCE)
```

Comment:
Thanks for all the fixes. Still, make clear that license is MIT in readme, so people don't need to look into license file just to know that. So make it `Released under the MIT [License](LICENSE)`. - note the typo in file name.